client/js/widgets/widget.js

```diff
@@ -93,17 +95,25 @@ export class Widget extends StateManaged {
     this.domElement.addEventListener('mouseleave',  e => this.hideEnlarged(e), false);
     this.domElement.addEventListener("touchstart", e => this.touchstart(), false);
     this.domElement.addEventListener("touchend", e => this.touchend(), false);
+    this.domElement.addEventListener('mousedown', e => this.selected(), false);
+    this.domElement.addEventListener('mouseup', e => this.selectedEnd(), false);
```

Comment:
Oh, and in general I don't like these event handlers. The others were added at some point to improve `enlarge` but I want to redo that at some point. All the mouse/touch stuff of widgets is usually handled by `mousehandling.js`. That's definitely not a deal breaker for this PR if it works though.